### DISCUSSION TITLE
Lunr JS search support

### DISCUSSION THREAD

**Discussion by AjaniBilby** [requarks/wiki#5915] [Help / Questions] [answered]
I know you already have quite a lot of features already in the pipeline for new search engines let alone other features. However I'd like to recommend adding [Lunr](https://lunrjs.com/) support if possible; as it is a open source search engine written in JS which would then have easy drop in support depending on how your backend is written.

It's used in [Material.io MkDocs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/) as an example of it in another open source similar project.

**Comment by NGPixel** [comment, marked as answer]
External search engines are being deprecated in v3.
